docs/faststore/docs/customization/building-sections/override-use-cases/adding-an-image-to-the-alert-component.mdx

```diff
@@ -0,0 +1,154 @@
+      - The override options specify:
+        - **`Section`:** The section to be overridden (Alert).
+        - **`components`:** An object containing the override for the Icon component.
+
+    - Inside the Icon override:
+      - **`Component`:** A function returning an `Image` component. This displays the image specified by the `src` prop.
+      - **`width`** and **`height`:** Optional props to set the size of the image.
+
+    - The use of `useMemo` ensures this override is only created once, improving performance.
+    - The function returns the `OverriddenAlert` component, spreading the `otherProps` and setting the `icon` prop to `noop` (effectively disabling the default icon behavior).
+
+    > ℹ️ For further details on code implementation, see the `AlertWithImage` folder available in the playground.store repository.
+
+6. Add the section to the Headless CMS by following the instructions available for sending components to the Headless CMS.
+
+The following schema was used as an example:
+
+    ```json 
+    [
+        {
+        "name": "AlertWithImage",
+        "schema": {
+        "title": "Alert with image",
+        "description": "Add an Alert that has an Image as an icon",
+        "type": "object",
+        "required": ["src", "content", "dismissible"],
+        "properties": {
+            "src": {
+            "type": "string",
+            "title": "Image source",
+            "description": "Source of the Alert icon image"
+            },
+            "content": { "type": "string", "title": "Content" },
+            "link": {
+            "title": "Link",
+            "type": "object",
+            "properties": {
+                "text": { "type": "string", "title": "Link Text" },
+                "to": { "type": "string", "title": "Action link" }
+            }
+            },
+            "dismissible": {
+            "type": "boolean",
+            "default": false,
+            "title": "Is dismissible?"
+            }
+          }
+        }
+      }
+    ]
+    ```
+
+> ℹ️ For further details on code implementation, see the `sections.json` file available in the playground.store repository.
+
+## Results
+
+Once you have [set your development mode](https://developers.vtex.com/docs/guides/faststore/headless-cms-previewing-changes-in-development-mode) to see the changes locally, you can see the image as an icon in the alert:
+
+- **Before**
+  
+  The native Alert displays one of the default icons, the `BellRinging`, following the sentence `Get 15% off today: NEW15! Buy now`.
+
+  ![before- alert-with-image](https://vtexhelp.vtexassets.com/assets/docs/src/before-custom-alert-image___fbd2316b0a78c7756fcc8523e7e6b181.png) 
+  
+- **After**
+  
+  The native Alert was replaced with the custom one, which features a pink image in the form of an icon added through Headless CMS.
+  
+  ![after - alert-with-image](https://vtexhelp.vtexassets.com/assets/docs/src/after-custom-alert-image___c4f735821a289a0fd0ce72947371e154.png)
```

Comment:
in my opinion, the banner catches too much attention. it took me a while to see the alert on top and see the icon. i would suggest cropping the image to get only the header and the alert.

Reply:
Makes sense, and I adjusted the image.